Homework 3/Homework3-1.py

```diff
@@ -0,0 +1,13 @@
+# Задание: Найти самое длинное слово в введенном предложении.
+# Учтите что в предложении есть знаки препинания.
+
+x = input('Введите строку: ')
+x = x.split()
+for i in range(0, len(x)):
+    x[i] = x[i].strip(',.!?')
```

Comment:
`. , ; :  ( ) [ ]` are also valid chars in sentence. 
Please note that space can be missed: `work(next word`